dkpro-statistics-agreement/src/main/java/org/dkpro/statistics/agreement/visualization/UnitizingMatrixPrinter.java

```diff
@@ -90,7 +90,7 @@ public void print(final PrintStream out, final IUnitizingAnnotationStudy study,
             annotations1[i] = ' ';
         }
         for (IUnitizingAnnotationUnit unit : study.getUnits()) {
-            if (unit.getRaterIdx() == rater1 && unit.getCategory() == category) {
+            if (unit.getRaterIdx() == rater1 && unit.getCategory().equals(category)) {
```

Comment:
Might be better to use `Objects.equals(...)` here to avoid potential NullPointerExceptions?

Reply:
Indeed. I'll update